include_all_in_one/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# This script will create the amalgamated library into 
```

Comment:
Please inline this into the root CML file, prefer not to fragment it if not necessary. When inlining, follow the style existing style, i.e. create a new section like `# ---- Declare include_all_in_one ----`